pkg/k8sutil/k8sutil.go

```diff
@@ -157,3 +160,18 @@ func CreateOrUpdateEndpoints(eclient clientv1.EndpointsInterface, eps *v1.Endpoi
 
 	return nil
 }
+
+// GetMinorVersion returns the minor version as an integer
+func GetMinorVersion(dclient discovery.DiscoveryInterface) (int, error) {
+	v, err := dclient.ServerVersion()
+	if err != nil {
+		return 0, nil
```

Comment:
I think it's fine to rely on properly compiled Kubernetes, and crash if it's not. From your comment I read that on latest minikube releases it works?